serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/service/Orchestration.scala

```diff
@@ -64,29 +67,41 @@ trait Orchestration extends RestClient with LazyLogging with SprayJsonSupport wi
       )
       postRequest(apiUrl("api/billing"), Map("projectName" -> projectName, "billingAccount" -> billingAccount))
 
+      // QA-1759: Not going to update org.broadinstitute.dsde.workbench.service.BillingProject
+      // because I don't want to update every use of that record just so I can log the project
+      // creation error message here. Please use `RawlsBillingProject` from Rawls instead.
+      case class YetAnotherBillingProject(projectName: String,
+                                          creationStatus: BillingProjectStatus,
+                                          message: Option[String])
+
       Retry.retry(10.seconds, 20.minutes) {
-        Try(responseAsList[String](parseResponse(getRequest(apiUrl("api/profile/billing"))))) match {
+        Try(parseResponse(getRequest(apiUrl(s"api/billing/v2/$projectName")))) match {
           case Success(response) =>
-            response
-              .map { p =>
-                BillingProject(p("projectName"),
-                               BillingProjectRole.withName(p("role")),
-                               BillingProjectStatus.withName(p("creationStatus"))
-                )
-              }
-              .find(p => p.projectName == projectName && BillingProjectStatus.isTerminal(p.creationStatus))
-          case Failure(t) => logger.info(s"Billing project creation encountered an error: ${t.getStackTrace}"); None
+            val project = mapper.readValue(response, classOf[Map[String, String]])
+            YetAnotherBillingProject(
+              projectName = project("projectName"),
+              creationStatus = BillingProjectStatus.withName(project("creationStatus")),
```

Comment:
just tried the v2 get BP endpoint and I think the key is `status` and not `creationStatus`.


```
{
  "billingAccount": "billingAccounts/00708C-45D19D-27AAFA",
  "invalidBillingAccount": false,
  "projectName": "yet-another-v2-bp",
  "roles": [
    "Owner"
  ],
  "status": "Ready"
}

```
https://rawls.dsde-dev.broadinstitute.org/#/billing_v2/getBillingProject

```suggestion
              creationStatus = BillingProjectStatus.withName(project("status")),
```